src/app/archive/page.tsx

```
@@ -2,56 +2,67 @@
-          <ul className="list-disc ml-4 space-y-1">
-            { item?.listInstruments.map( tool =>(
-              <li> {tool} </li>
-            ))}
-          </ul>
-        </div>
-         )}
+        {item.listInstruments?.length && (
+          <div className="border-l-2 border-amber-900 pl-4 my-4 font-mono text-sm">
+            <ul className="list-disc ml-4 space-y-1">
+              {item?.listInstruments.map(tool => (
+                <li> {tool} </li>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)